integration_tests/test_fine_tunes.py

```diff
@@ -19,6 +20,10 @@
 MAX_RETRIES = 10
 
 
+@pytest.mark.skipif(
+    os.getenv("ENV") == "prod",
```

Comment:
skip if == "launch" as well?maybe we can limit this to circleci for now

Reply:
training cluster is proper set up for this. i do think we want some e2e test like this.